### DISCUSSION TITLE
在主站内文章页面如何根据请求接口返回内容渲染数据

### DISCUSSION THREAD

**Discussion by Kimser** [halo-dev/halo#5535] [Q&A]
### Your current Halo version

2.13

### Describe this feature

文章内的数据需要从接口中获取然后需然在表格中 目前不清楚需要如果做 

### Additional information

文章内的数据需要从接口中获取然后需然在表格中 目前不清楚需要如果做

**Comment by ruibaby** [comment]
可以使用 `/apis/api.content.halo.run/v1alpha1/posts/{name}` 接口，其中有 content 字段。

文档：https://demo.halo.run/webjars/swagger-ui/index.html#/api.content.halo.run%2Fv1alpha1%2FPost

可以简单描述一下你的用途吗？

/kind support

**Comment by Kimser** [comment]
首先在console后台创建一个歌单页面
之后在这个页面有个 歌单列表的Table显示我所有的歌单
问题1：如何请求接口
问题 2：如何写 html table tab 之类的样式 、根据接口数据渲染
@ruibaby

**Comment by ruibaby** [comment]
> 首先在console后台创建一个歌单页面 之后在这个页面有个 歌单列表的Table显示我所有的歌单 问题1：如何请求接口 问题 2：如何写 html table tab 之类的样式 、根据接口数据渲染 @ruibaby

具体实现细节我们无法提供帮助。

此问题不属于 issue，我将转移到 Discussion。
